src/content/docs/setup/analytics/instrumentation.mdx

```diff
@@ -4,10 +4,15 @@ description: Learn how to instrument your Adobe Commerce on Edge Delivery Servic
 ---
 
 import OptionsTable from '@components/OptionsTable.astro';
+import Aside from '@components/Aside.astro';
 
+User interaction events power Adobe Sensei features, like intelligent merchandising for Live Search and to personalize product recommendations units. These events also populate performance dashboards for [Live Search](https://experienceleague.adobe.com/en/docs/commerce/live-search/live-search-admin/performance).
```

Comment:
for [Live Search] and product recommendations https://experienceleague.adobe.com/en/docs/commerce/product-recommendations/admin/workspace#data-collection